reporthandling/apis/severity.go

```diff
@@ -30,15 +38,15 @@ func SeverityNumberToString(severityNumber int) string {
 	*/
 	switch severityNumber {
 	case SeverityCritical:
-		return "Critical"
+		return SeverityCriticalString
 	case SeverityHigh:
-		return "High"
+		return SeverityHighString
 	case SeverityMedium:
-		return "Medium"
+		return SeverityMediumString
 	case SeverityLow:
-		return "Low"
+		return SeverityLowString
 	default:
-		return "Unknown"
+		return SeverityUnknownString
```

Comment:
I took the liberty of extracting this to a constant, so we can avoid hardcoding values.